packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -181,47 +227,52 @@ const RecentDiscussionSubscribeReminder = ({classes}: {
     </AnalyticsWrapper>
   } else if (!currentUser || adminBranch===0) {
     // Not logged in. Show a create-account form and a brief pitch.
-    return <AnalyticsWrapper branch="logged-out">
+    const subscribeTextNode = forumTypeSetting.get() === 'EAForum' ? eaForumSubscribePrompt : (
       <div className={classes.message}>
         To get the best posts emailed to you, create an account! {subscriptionDescription}
       </div>
+    );
+    return <AnalyticsWrapper branch="logged-out">
+      {subscribeTextNode}
       <div className={classes.loginForm}>
         <WrappedLoginForm startingState="signup" />
       </div>
       {adminUiMessage}
-      <div className={classes.buttons}>
-        {maybeLaterButton}
-        {dontAskAgainButton}
-      </div>
     </AnalyticsWrapper>
   } else if (!userHasEmailAddress(currentUser) || adminBranch===1) {
```

Comment:
This branch has some remaining bugs, such as the `SignupSubscribeToCurated` component always setting `subscribeChecked` to `true` even if the user unchecks it. I'm not sure how often this case comes up for your userbase, but it might be worth testing and fixing this.